go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/hashicorp/terraform-plugin-framework-validators v0.12.0
 	github.com/hashicorp/terraform-plugin-go v0.19.0
 	github.com/hashicorp/terraform-plugin-testing v1.5.1
-	github.com/infobloxopen/bloxone-go-client v0.0.0-20231101055206-41e81034eeb1
+	github.com/infobloxopen/bloxone-go-client v0.0.0-20231106185847-34317d911a03
```

Comment:
I have merged the changes to the master and didn't delete the work branch yet, will do once you update the PR with the newer version from master

Reply:
updated to master version